src/lib/coapcore/src/seccontext.rs

```diff
@@ -163,6 +166,26 @@ enum SecContextStage<Crypto: lakers::Crypto> {
     Oscore(liboscore::PrimitiveContext),
 }
 
+#[cfg(feature = "defmt")]
+impl<Crypto: lakers::Crypto> defmt::Format for SecContextStage<Crypto> {
+    fn format(&self, f: defmt::Formatter) {
+        match self {
+            SecContextStage::Empty => defmt::write!(f, "Empty"),
+            SecContextStage::EdhocResponderProcessedM1 { c_r, .. } => {
+                defmt::write!(f, "EdhocResponderProcessedM1 {{ c_r: {:?}, ... }}", c_r)
```

Comment:
Those curly braces were introduced by rustfmt; for consistency with the non-curly-braces version for Empty, there should not be semicolons, so that in all cases the match evaluates to whatever defmt::write! returns (probably `()` but I'm not sure).